package.json

```diff
@@ -69,8 +74,7 @@
     "react-to-print": "^2.0.0-alpha-6",
     "react-visibility-sensor": "^5.0.2",
     "simpl-schema": "^1.12.0",
-    "slick-carousel": "^1.8.1",
-    "styled-components": "^5.2.1"
+    "slick-carousel": "^1.8.1"
   },
```

Comment:
just a question,
is this package.json totally up to date
are all these packages are in absolute use
like loadable/components or babel-polyfill babel-runtime or etc
so from antjs or goormetjs or anything else used in passed .... 
can we check and make this file up to date if needed

Reply:
Good questions! 

To my knowledge they are all in use but not so sure honestly with those that are babel stuff. Probably there are those we don't need, like we have both `@babel/runtime` and `babel-runtime` and I have no clue what's their differences... This could in itself be a new PR topic to optimize actually. Then it's probably be good to do some updates too. 

And then no, definitely not all of them are up to date. It's a hard task to keep them all updated in my experience, mostly because things tend to break often when updated...